Makefile

```diff
@@ -30,12 +30,21 @@ nss: test
 	$(MAKE) nss-production -C test
 	@echo $(CYAN)"\nDone ! Generated code can be found in 'snapshots/nss-production'."$(NORMAL)
 
+dependencies:
+	@echo $(CYAN)"# Get and build F* and KreMLin"$(NORMAL)
+	opam switch 4.04.2
+	eval `opam config env`
+	git submodule update --init
+	opam config exec -- make -C dependencies/FStar/src/ocaml-output
```

Comment:
You need to add the following line here.
`opam config exec -- make -C dependencies/FStar/ulib/ml`